src/sprite_store.cpp

```diff
@@ -458,20 +445,19 @@ bool SpriteManager::LoadPATH(RcdFileReader *rcd_file, const ImageMap &sprites)
 		case  8: pt = PAT_TILED; break;
 		case 12: pt = PAT_ASPHALT; break;
 		case 16: pt = PAT_CONCRETE; break;
-		default: return false; // Unknown type of path.
+		default: rcd_file->Error("Invalid path type");
 	}
 
 	uint16 width = rcd_file->GetUInt16();
 	rcd_file->GetUInt16(); /// \todo Remove height from RCD block.
 
 	SpriteStorage *ss = this->GetSpriteStore(width);
-	if (ss == nullptr) return false;
+	if (ss == nullptr) rcd_file->Error("String storage not found");
```

Comment:
```suggestion
	if (ss == nullptr) rcd_file->Error("Sprite storage not found");
```
You may want to check all `GetSpriteStorage` calls, I may have missed some.